public_html/wp-content/mu-plugins/blocks/source/utils/render-frontend.js

```diff
@@ -92,6 +92,10 @@ export default ( selector, Block, getProps = getPropsFallback ) => {
 				/>,
 				element
 			);
+
+			// todo-front why pulling render() from react-dom instead of @wordpress/element ?
+			// make sure that react-dom, lodash, etc aren't getting bundled into the build, should be using from core versions as external instead
+				// seems lke they're being bundled live-schedule.min.js
```

Comment:
Happened to see this in my git app — it's meant to save from enqueuing all of `wp-element` on the frontend, it should not affect build since since both `react-dom` and `wp-element` are externals (the only change is to the dependency list in the asset.php file).

however, the block in the live-schedule uses wp-element, so it's added to the deps list anyway, so this is kind of a pointless "optimization" 🤷🏼‍♀️  Since I see you're already importing from `wp-element` in this file, you can just combine them.